--- a/net/minecraft/world/item/Items.java
+++ b/net/minecraft/world/item/Items.java
@@ -123,6 +123,7 @@ public class Items {
     public static final Item ENDER_PEARL;
     public static final Item BLAZE_ROD;
     public static final Item GHAST_TEAR;
+    public static final Item LEAPS_INGOT;
     public static final Item GOLD_NUGGET;
     public static final Item NETHERITE_INGOT;
     public static final Item NETHERITE_SCRAP;
@@ -789,6 +790,7 @@ public class Items {
         ENDER_PEARL = registerItem("ender_pearl", new EnderPearlItem(new Item.Properties()));
         BLAZE_ROD = registerItem("blaze_rod", new Item(new Item.Properties()));
         GHAST_TEAR = registerItem("ghast_tear", new Item(new Item.Properties()));
+        LEAPS_INGOT = registerItem("leaps_ingot", new Item(new Item.Properties()));
         GOLD_NUGGET = registerItem("gold_nugget", new Item(new Item.Properties()));
         NETHERITE_INGOT = registerItem("netherite_ingot", new Item(new Item.Properties()), (new Item.Properties()).fireResistant());
         NETHERITE_SCRAP = registerItem("netherite_scrap", new Item(new Item.Properties()), (new Item.Properties()).fireResistant());
